src/test-integration/java/dev/openfga/sdk/example/Example1.java

```diff
@@ -0,0 +1,198 @@
+        models = fgaClient.readAuthorizationModels().get();
+        System.out.println("Models Count: " + models.getAuthorizationModels().size());
+
+        // ReadLatestAuthorizationModel - after Write
+        var latestAuthorizationModel = fgaClient.readLatestAuthorizationModel().get();
+        System.out.println("Latest Authorization Model ID "
+                + latestAuthorizationModel.getAuthorizationModel().getId());
+
+        // Set the model ID
+        fgaClient.setAuthorizationModelId(
+                latestAuthorizationModel.getAuthorizationModel().getId());
+
+        // Write
+        System.out.println("Writing Tuples");
+        fgaClient
+                .write(
+                        new ClientWriteRequest()
+                                .writes(List.of(new ClientTupleKey()
+                                        .user("user:anne")
+                                        .relation("writer")
+                                        ._object("document:roadmap"))),
+                        new ClientWriteOptions()
+                                .disableTransactions(true)
+                                .authorizationModelId(authorizationModel.getAuthorizationModelId()))
+                .get();
+        System.out.println("Done Writing Tuples");
+
+        // Read
+        System.out.println("Reading Tuples");
+        var readTuples = fgaClient.read(new ClientReadRequest()).get();
+        System.out.println("Read Tuples" + mapper.writeValueAsString(readTuples));
+
+        // ReadChanges
+        System.out.println("Reading Tuple Changess");
+        var readChangesTuples =
+                fgaClient.readChanges(new ClientReadChangesRequest()).get();
+        System.out.println("Read Changes Tuples" + mapper.writeValueAsString(readChangesTuples));
+
+        // Check
+        System.out.println("Checking for access");
+        try {
+            var failingCheckResponse = fgaClient
+                    .check(new ClientCheckRequest()
+                            .user("user:anne")
+                            .relation("reader")
+                            ._object("document:roadmap"))
+                    .get();
+            System.out.println("Allowed: " + failingCheckResponse.getAllowed());
+        } catch (Exception e) {
+            System.out.println("Failed due to: " + e.getMessage());
+        }
+
+        // Checking for access with context
+        // TODO: Add ClientCheckRequest.context
+        // System.out.println("Checking for access with context");
+        // var checkResponse = fgaClient
+        //         .check(new ClientCheckRequest()
+        //                 .user("user:anne")
+        //                 .relation("reader")
+        //                 ._object("document:roadmap")
+        //                 .context(Map.of("ViewCount", 100)))
+        //         .get();
+        // System.out.println("Allowed: " + checkResponse.getAllowed());
```

Comment:
Making the fix here: https://github.com/openfga/java-sdk/pull/41

I didn't include the change in this PR since it will need a release first for the example to "just work"